eng/ci/templates/official/jobs/run-benchmarks.yml

```diff
@@ -55,6 +58,9 @@ jobs:
   - script: dotnet tool install -g Microsoft.Crank.Agent --version "0.2.0-*"
```

Comment:
Is the crank agent still needed on this job? Or is only the controller needed?

Reply:
yes. We are using this agent for load generation as well. (If we want to create an agent for that, we could, but I chose this approach for now). So agent is needed.